src/slurm_plugin/cluster_event_publisher.py

```diff
@@ -509,6 +517,52 @@ def publish_node_launch_events(self, failed_nodes: Dict[str, List[str]]):
             event_supplier=self._flatten_failed_launch_nodes(failed_nodes),
         )
 
+    @log_exception(logger, "publish_job_info_events", catch_exception=Exception, raise_on_error=False)
+    def publish_job_events(self, jobs: List[SlurmJobInfo]):
+        """Publish events for job information."""
+        timestamp = ClusterEventPublisher.timestamp()
+        self.publish_event(
+            logging.DEBUG,
+            **SCONTROL_SHOW_JOB_COUNT,
+            timestamp=timestamp,
+            event_supplier=self._job_count_supplier(jobs),
+        )
+        for job in jobs:
+            self.publish_event(
+                logging.INFO,
+                **SCONTROL_SHOW_JOB_INFORMATION,
+                timestamp=timestamp,
+                event_supplier=self._job_info_supplier(job),
+            )
+
+    # Example event generated by this function:
```

Comment:
Could you move the example to above the method definition?

Reply:
sure, changed